pkg/controller/bootstrap/testdata/bootstrap/image-content-source-policy-0.yaml

```diff
@@ -7,4 +7,4 @@ spec:
   repositoryDigestMirrors:
   - mirrors:
     - registry.mirror.example.com/ocp
-    source: registry.product.example.org/ocp/4.2-DATE-VERSION
+    source: registry.product.example.org/ocp/4.2-date-version
```

Comment:
Hmm, does this actually change anything? Just curious

Reply:
These files are also used in the e2e test. To fix the error I saw in the ci e2e test that is image repository name must be lowercase.